### PR TITLE
Add name field for status command

### PR DESCRIPTION
**Pull request by hanzei**
**before**
Screenshot from 2024-09-30 10-07-37

**after**
Screenshot from 2024-09-30 10-08-47

### CODE REVIEW

**Comment by hanzei** [comment]
**No not merge**

I need to fix the README

File: model/campaign.go [hunk 1]
```diff
@@ -37,6 +38,20 @@ type Campaign struct {
 	Tickets        []*Ticket    `json:"tickets,omitempty"`
 }
 
+func NewCampaign(name string) *Campaign {
+	return &Campaign{
+		Tickets: []*Ticket{},
```
Comment (mgdelacroix) [inline model/campaign.go:43]:
```suggestion
	return &Campaign{
		Name: name,
		Tickets: []*Ticket{},
```

File: README.md [hunk 2]
```diff
@@ -41,6 +41,7 @@ Once we have both files, we can run `campaigner init` to create the campaign. Th
 
 ```sh
 $ campaigner init \
+    -- name Remove ToDo comments
```
Comment (mgdelacroix) [inline README.md:44]:
```suggestion
    --name Remove ToDo comments
```

**Review by mgdelacroix — APPROVED** [review]
LGTM appart from the suggestion

**Pull request merged**